run_tests.sh

```diff
@@ -289,7 +289,9 @@ then
        shift
     fi
     MY_UID=$(id -u)
-    DOCKER_RUN_EXTRA_ARGS="-e GALAXY_TEST_UID=${MY_UID} ${DOCKER_RUN_EXTRA_ARGS}"
+    # Skip client build process in the Docker container for all tests, the Jenkins task builds the client
+    # locally before testing - you will need to do this also if using this script for Selenium testing.
+    DOCKER_RUN_EXTRA_ARGS="-e GALAXY_TEST_UID=${MY_UID} -e GALAXY_SKIP_CLIENT_BUILD=1 ${DOCKER_RUN_EXTRA_ARGS}"
```

Comment:
Is this still needed now that https://github.com/galaxyproject/galaxy/pull/5679 has been merged?

Reply:
I believe so - here is the full console log on that PR for the API tests - https://jenkins.galaxyproject.org/job/docker-api/10848/consoleFull. If you compare that to this PR's API tests https://jenkins.galaxyproject.org/job/docker-api/10879/consoleFull you can see that after #5679 API tests (and framework tests) were still building the client but they don't anymore with this PR.